src/io/sb3/fromSb3.ts

```diff
@@ -336,7 +284,7 @@ function getBlockScript(blocks: { [key: string]: sb3.Block }) {
   return blockWithNext;
```

Comment:
This may be outside the scope of this PR (if so, I'll create an issue to track it separately).

It looks like the `blockWithNext` function is recursive, and creates a new `Array` object *every time* it recurses. This sounds like extremely bad news for performance, especially with long scripts. Could it be changed to be less "clever" and use an iterative approach?